models/solution/activity.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright © Mapotempo, 2021
+#
+# This file is part of Mapotempo.
+#
+# Mapotempo is free software. You can redistribute it and/or
+# modify since you respect the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# Mapotempo is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the Licenses for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Mapotempo. If not, see:
+# <http://www.gnu.org/licenses/agpl.html>
+#
+require './models/base'
+
+module Models
+  class RouteActivity < Base
+    # field :point_id
+    field :type
+    field :alternative
+    field :reason, default: nil
+    # TODO: The following fields should be merged in v2
```

Comment:
What do you think about starting to return _id_ field already ?

Reply:
As we are not in v2, we still need to use these multiple ids. But yes, at the end we definitely need to use a single id field

Reply:
I meant, can't we return both _id_ and _service_id_/_pickup_id_/_shipment_id_ fields at the same time ? That would help anticipating transition to v2, because new customers could use _id_ field. We will have a transition period anyways.